lua/telescope/utils.lua

```diff
@@ -370,6 +379,11 @@ utils.transform_path = function(opts, path)
   end
 end
 
+utils.calc_result_length = function()
```

Comment:
lets make this local

Reply:
Agreed, was originally trying to calculate the length elsewhere but couldn't get it to work properly.
Forgot to make it local again 🤦